crypto/internal.h

```diff
@@ -894,6 +895,28 @@ static inline void CRYPTO_store_word_le(void *out, crypto_word_t v) {
 
 
 // FIPS functions.
+#if defined(AWSLC_FIPS)
+enum fips_approved_algorithm_t {
```

Comment:
I wonder if we should make them capital. Is there a convention for enums in the code?

Reply:
Changed to capital letters